core/src/actors/chain_manager/mod.rs

```diff
@@ -341,7 +341,7 @@ impl ChainManager {
         for transaction in self.transactions_pool.iter() {
             // Currently, 1 weight unit is equivalent to 1 byte
             let transaction_weight = transaction.size();
-            let transaction_fee = transaction.fee();
+            let transaction_fee = transaction.fee(&self.transactions_pool).unwrap();
```

Comment:
I'm concerned about this `unwrap`.

Reply:
The issue is that I changed the `fee` function signature, before it returned `u64`, now it returns `Result<u64, ...>`. I'm using unwrap to avoid changing this function signature too, because that would lead to a bigger refactor and maybe is better to address it later.

Reply:
Ok, then let's put a FIXME here :+1: